### PR TITLE
fix(lambda.js): catch unhandled promise rejections

### PR DESCRIPTION
**Pull request by ronnathaniel**

### CODE REVIEW

File: src/wrappers/lambda.js [hunk 1]
```diff
@@ -111,6 +114,17 @@ function baseLambdaWrapper(
...
         };
+        // catch an unhandled promise rejection within the user's code
+        process.once('unhandledRejection', (reason) => {
+            utils.debugLog('Unhandled Promise Rejection caught. Please handle the rejected promise');
+            const rejectedError = {
+                name: 'UnhandledPromiseRejectionError',
+                message: reason || '',
+                stack: '',
+            };
+            eventInterface.setException(runner, rejectedError, false);
```
Comment (henperez) [inline src/wrappers/lambda.js:122]: Is this still needed?

Comment (ronnathaniel) [reply src/wrappers/lambda.js:122]: If/when we add an earlier detection, this will set to show on dashboard

Comment (ronnathaniel) [reply src/wrappers/lambda.js:122]: We can take it out now, but will have to be added in at that point

File: src/wrappers/lambda.js [hunk 2]
```diff
@@ -10,7 +10,9 @@ const { getConfig } = require('../config.js');
 const awsLambdaTrigger = require('../triggers/aws_lambda.js');
 const eventInterface = require('../event.js');
 const lambdaRunner = require('../runners/aws_lambda.js');
-const { STEP_ID_NAME, MAX_VALUE_CHARS, EPSAGON_EVENT_ID_KEY } = require('../consts.js');
+const {
```
Comment (henperez) [inline src/wrappers/lambda.js]: Not needed

File: src/wrappers/lambda.js [hunk 3]
```diff
@@ -97,6 +99,7 @@ function baseLambdaWrapper(
             runner.setDuration(utils.createDurationTimestamp(startTime));
         };
 
+
```
Comment (henperez) [inline src/wrappers/lambda.js]: Same

File: src/wrappers/lambda.js [hunk 4]
```diff
@@ -295,6 +309,8 @@ function baseLambdaWrapper(
                         return returnValue;
                     });
             }
+
```
Comment (henperez) [inline src/wrappers/lambda.js]: Same

**Comment by henperez** [comment]
Overall looks good 💯  - Small changes

**Review by tvaintrob — APPROVED** [review]

**Pull request merged**